addons/mail/static/src/core/common/thread.js

```diff
@@ -78,7 +80,11 @@ export class Thread extends Component {
         this.present = useRef("load-newer");
         this.messagesRef = useRef("messages");
         this.loadOlderState = useVisible("load-older", () => {
-            if (this.loadOlderState.isVisible && !this.isJumpingRecent) {
+            if (
+                this.loadOlderState.isVisible &&
+                !this.isJumpingRecent &&
+                !this.env.messageHighlight?.highlightedMessageId
```

Comment:
Again I don't get why `highlightedMessageId` is used. At a glance looks incorrect way to fix supposedly a bug?